lib/rules/jsx-closing-bracket-location.js

```diff
@@ -274,7 +275,7 @@ module.exports = {
               case 'tag-aligned':
               case 'line-aligned':
                 return fixer.replaceTextRange([cachedLastAttributeEndPos, node.range[1]],
-                  `\n${getIndentation(tokens, expectedLocation, correctColumn)}${closingTag}`);
+                  `\n${getIndentation(tokens, expectedLocation, correctColumn || 0)}${closingTag}`);
```

Comment:
can the signature of `getIndentation` instead be changed so that the last arg is optional?

Reply:
updated